cli/src/pcluster/templates/cluster_stack.py

```diff
@@ -1214,13 +1213,6 @@ def _add_head_node(self):
                                 if head_node.disable_simultaneous_multithreading_manually
                                 else "false",
                                 "CloudFormationUrl": cloudformation_url,
-                                "DisableSudoAccessForDefaultUserConfig": get_cloud_config_for_default_user(
-                                    self.config.disable_sudo_access_default_user
-                                ),
-                                "OSUser": OS_MAPPING[self.config.image.os]["user"],
-                                "DisableSudoAccessForDefault": "true"
-                                if self.config.disable_sudo_access_default_user
-                                else "false",
```

Comment:
Do we need to keep this?

Reply:
No we dont need to keep this section at all. This section plugs the values in the Keys for `headnode/user-data.sh` which is basically changing the Launch Template for headNode. 

I want this value to be added in Dna.json which is what I am doing here https://github.com/aws/aws-parallelcluster/pull/6004/files#diff-ca8ebe7ee87b21621356e66a7385c9256d458ddd3415b817c62de31132bb2327:~:text=).lower()%2C-,%22disable_sudo_access_for_default_user%22%3A%20%22true%22,**(,-get_slurm_specific_dna_json_for_head_node(self